### ISSUE TITLE
Normalize before differential accessibility between conditions

### ISSUE THREAD

**Issue by hhvu0102** [timoast/signac#373]
Hello,

I hope you can help explain how you normalize between 2 conditions before doing differential accessibility analysis.
I understand we need to integrate data if there's a batch effect, then follow the tutorial: https://satijalab.org/seurat/v3.2/immune_alignment.html. Also, according to issue #222 I should use `'peaks'` assay. However, when building `'peaks'` assay (as in https://satijalab.org/signac/articles/mouse_brain_vignette.html), I presume there's no normalization step. Am I understanding this wrong? And if not, how should be normalize before differential accessibility analysis?
Thanks a lot!

**Comment by timoast** [comment]
The differential accessibility test uses the TF-IDF values, which incorporates a per-cell depth normalization step. If you are comparing between multiple batches, you can also include batch as a latent variable in the test.

**Comment by hhvu0102** [comment]
Hi Tim,

Thank you very much for your reply. I only have 2 batches (i.e. 2 conditions) right now, but can you elaborate how to put batch as a latent variable? Does this mean we would build a meta information column in our `@meta.data`?

**Comment by timoast** [comment]
To incorporate batch as a latent variable in the model, you can pass the `latent.vars` argument in `Seurat::FindMarkers()`. This should be the name of a metadata column in the object that identifies the different batches. See the docs for FindMarkers for more information.

**Comment by hhvu0102** [comment]
Thank you!
